conjure-go-client/httpclient/client_builder.go

```diff
@@ -161,27 +161,40 @@ func newClient(ctx context.Context, b *clientBuilder, params ...ClientParam) (Cl
 	middleware := b.HTTP.Middlewares
 	b.HTTP.Middlewares = nil
 
-	httpClient, err := b.HTTP.Build(ctx)
+	httpClient, validParams, err := b.HTTP.Build(ctx, config, reloadErrorSubmitter)
 	if err != nil {
 		return nil, err
 	}
 
+	if !b.AllowEmptyURIs {
+		// Validate that the URIs are not empty.
+		if curr := validParams.CurrentValidatedClientParams(); len(curr.URIs) == 0 {
+			return nil, werror.ErrorWithContextParams(ctx, "httpclient URLs must be set in configuration or by constructor param",
```

Comment:
nit, but if all exported config/param names use "URI", error message should probably also use "URI" rather than "URL" for consistency.